toolchain/lower/file_context.cpp

```diff
@@ -81,6 +84,15 @@ auto FileContext::Run() -> std::unique_ptr<llvm::Module> {
 
   // Specific functions are lowered when we emit a reference to them.
   specific_functions_.resize(sem_ir_->specifics().size());
+  // Additional data stored for specifics, for when attempting to coalesce.
+  // Indexed on generic_id
+  lowered_specifics_.resize(sem_ir_->generics().size());
+  // Indexed on specific_id
```

Comment:
Punctuation and type (rather than value).

```suggestion
  // Indexed by `GenericId`.
  lowered_specifics_.resize(sem_ir_->generics().size());
  // Indexed by `SpecificId`.
```